docs/_static/spec/rasa.yml

```diff
@@ -296,6 +296,57 @@ paths:
         500:
           $ref: '#/components/responses/500ServerError'
 
+  /conversations/{conversation_id}/inject-intent:
+    post:
+      security:
+        - TokenAuth: []
+        - JWT: []
+      operationId: injectConversationIntent
+      tags:
+        - Tracker
+      summary: Inject an intent into a conversation
+      description: >-
+        Sends a message with the specified intent and entities in place
+        of the user, then predicts and executes a response action.
```

Comment:
Can you send an actual user message, or just intent and entities?

Reply:
Just intent and entities. I updated the text for clarity.

Reply:
Okay. makes sense bc you want to trigger an intent anyway and don't want it left up to NLU 👍